data/src/test/java/camp/nextstep/edu/github/data/GithubApiServiceTest.kt

```diff
@@ -0,0 +1,51 @@
+package camp.nextstep.edu.github.data
+
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runTest
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.Before
+import org.junit.Test
+import org.junit.jupiter.api.assertAll
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.create
+import java.io.File
+
+@ExperimentalCoroutinesApi
+class GithubApiServiceTest {
+
+    private lateinit var server: MockWebServer
+    private lateinit var apiService: GithubApiService
+
+    @Before
+    fun setUp() {
+        server = MockWebServer()
+        apiService = Retrofit.Builder()
+            .baseUrl(server.url("/"))
+            .addConverterFactory(GsonConverterFactory.create())
+            .build()
+            .create()
+    }
+
+    @Test
+    fun 저장소를_불러올_수_있어야_한다() = runTest {
+        //given
+        val jsonPath = javaClass.getResource("/repository-200.json").path
+        val response = MockResponse()
+            .setBody(File(jsonPath).readText())
```

Comment:
```suggestion
            .setBody(File(jsonPath).readText())
            .setResponseCode(200)
```
응답코드가 200임을 표현하고 싶으시면, setResponseCode를 활용할 수 있어요.